src/components/Translation.tsx

```diff
@@ -1,35 +1,38 @@
-import htmr from "htmr"
+import htmr, { type HtmrOptions } from "htmr"
 import type { TOptions } from "i18next"
 import { useRouter } from "next/router"
 import { useTranslation } from "next-i18next"
 
 import { getRequiredNamespacesForPage } from "@/lib/utils/translations"
 
-import InlineLink from "./Link"
+import TooltipLink from "./TooltipLink"
 
 type TranslationProps = {
   id: string
   options?: TOptions
+  transform?: HtmrOptions["transform"]
 }
 
 // Custom components mapping to be used by `htmr` when parsing the translation
 // text
-const transform = {
-  a: InlineLink,
+const defaultTransform = {
+  a: TooltipLink,
 }
 
 // Renders the translation string for the given translation key `id`. It
 // fallback to English if it doesn't find the given key in the current language
-const Translation = ({ id, options }: TranslationProps) => {
+const Translation = ({ id, options, transform = {} }: TranslationProps) => {
   const { asPath } = useRouter()
   const requiredNamespaces = getRequiredNamespacesForPage(asPath)
 
   const { t } = useTranslation(requiredNamespaces)
   const translatedText = t(id, options)
 
   // Use `htmr` to parse html content in the translation text
-  // @ts-ignore
-  return htmr(translatedText, { transform })
+  return htmr(translatedText, {
+    // @ts-ignore
+    transform: { ...defaultTransform, ...transform },
```

Comment:
```suggestion
    transform: { ...defaultTransform, ...transform } as HtmrOptions["transform"],
```

Would this work to remove the ts-ignore?

Reply:
I revisited this and found the root problem (check [eb02aa2](https://github.com/ethereum/ethereum-org-website/pull/12236/commits/eb02aa2eaa260df75aaa1199a28b1a5235bf665c)). Basically, `htmr` was expecting some props from the Link component to be optional.